docs/develop/sdk/1-dev-setup.md

```diff
@@ -0,0 +1,150 @@
+---
+id: dev-setup
+title: Development Setup
+---
+import CodeBlock from '@theme/CodeBlock';
+import Example1 from '!!raw-loader!@site/code-examples/core_features/dev_setup.ts';
+
+If you want to develop a DApp, Wallet, web3 login or similar, you will need a running blockchain that doesn't require you to buy KILT just to develop and test the next big thing.
+For that you can use the peregrine network.
+The peregrine network is a parachain that is similar to the spiritnet (our mainnet) in functionality, but it's coin, the PILT, doesn't hold any monetary value.
+New features that we add to our blockchain, will first be tested on Peregrine before they are introduced to the Spiritnet.
+This gives you the chance to test your software with the newest blockchain features, before they are available on the mainnet.
+
+Even though the peregrine network is convenient, there are a scenarios where a public network is not the right choice for developers.
+If you need more funds than the faucet can provide, or if you need to reset the state of the blockchain, you will need to setup your own little KILT blockchain.
+
+Below, we will guide you through the process of connecting to the Peregrine network as well as running your own KILT blockchain.
+Moreover, we will explain the difference between our _Standalone_ and _Parachain_ runtimes Peregrine/Spiritnet.
+
+## Setup your Project
+
+Before we can connect to our blockchain, we need to create a small project that can connect to the chain.
+For that create a new directory and initialize a new project.
+We are also adding the KILT SDK, TypeScript and a couple of peer dependencies.
+
+```bash npm2yarn
+npm init -y
+npm install @kiltprotocol/sdk-js@0.27.0 \
+ ts-node \
+ typescript \
+ @types/node
+ @polkadot/keyring@8.7.1 \
+ @polkadot/util@8.7.1 \
+ @polkadot/util-crypto@8.7.1 \
+```
+
+After all these dependencies are added, we will add a simple script that prints the balance of a couple of well known development accounts.
+
+<CodeBlock className="language-js">
+  {Example1}
+</CodeBlock>
+
+Copy the above code into index.ts.
+You can know execute the script:
+
+```bash npm2yarn
+npm exec ts-node index.ts
+```
+
+The output should look like this:
+
+```
+Seed //Alice has account id 4siJtc4dYq2gPre8Xj6KJcSjVAdi1gmjctUzjf3AwrtNnhvy and 0
+Seed //Bob has account id 4r99cXtVR72nEr9d6o8NZGXmPKcpZ9NQ84LfgHuVssy91nKb and 0
+```
+
+None of the development accounts have usable balance on the public peregrine network.
```

Comment:
As I mentioned above, is there any specific reason to actually guide users through this for Peregrine, to then find out at the end that the accounts have no usable balance?

Reply:
Yes the reason is that i want to mention the dev accounts. But I also want to keep it simple. So instead of heaving two versions for peregrine and standalone I have one version for both and the reader learns that the dev accounts have funds on the local chain but not on peregrine.

Reply:
```suggestion
None of the development accounts have usable balance on the public Peregrine network.
```